app/src/main/java/io/scalac/degree/android/receiver/AlarmReceiver.java

```diff
@@ -28,12 +29,30 @@ public AlarmReceiver() {
     @SuppressLint("Wakelock")
     @Override
     public void onReceive(Context context, Intent intent) {
+        if (intent.getExtras().containsKey(NotificationsManager.EXTRA_NOTIFICATION_TYPE)) {
+            Logger.l("Alarm.forPost");
+            handlePostTalkEvent(intent);
+        } else {
+            Logger.l("Alarm.forNormal");
+            handleIncomingTalkEvent(intent);
+        }
+    }
+
+    private void handlePostTalkEvent(Intent intent) {
+        final String slotId = intent.getStringExtra(NotificationsManager.EXTRA_TALK_ID);
+
+        // TODO Texts probably will change.
+        notificationsManager.showPostNotification(slotId, "Give a vote!", "Add vote for the talk.");
```

Comment:
do stringow, jakiegos tlumaczenia nie bedzie? chyba ze to rzeczywiscie na chwile i testowo to olej